protocol/x/clob/keeper/liquidations.go

```diff
@@ -48,6 +54,8 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 	if numSubaccounts > 0 {
 		indexOffset = pseudoRand.Intn(numSubaccounts)
 	}
+
+	startGetLiquidationOrders := time.Now()
```

Comment:
Avoided using gauges for number of subaccounts liquidated / deleveraged since those are already added in the individual liquidation / deleveraging functions.